app/Filament/Resources/EmulatorResource.php

```diff
@@ -73,34 +73,46 @@ public static function infolist(Infolist $infolist): Infolist
                                 ->schema([
                                     Infolists\Components\TextEntry::make('name')
                                         ->label('Name')
-                                        ->helperText('Name of emulator'),
+                                        ->helperText('This is the name that will appear on the website.'),
 
                                     Infolists\Components\TextEntry::make('original_name')
                                         ->label('Original name')
-                                        ->helperText('Original name of emulator.'),
+                                        ->placeholder('none')
+                                        ->helperText("If this emulator is forked from another emulator, put the original emulator's name here. eg: 'Snes9x' is the original name for 'RASnes9x'."),
 
                                     Infolists\Components\TextEntry::make('description')
                                         ->label('Description')
-                                        ->helperText('Additional text to display on the download page.'),
+                                        ->placeholder('none')
+                                        ->helperText("This text used to be displayed on the Downloads page, but isn't anymore. You can add notes about the emulator here."),
```

Comment:
The fact that this used to be displayed on the downloads page seems unimportant. Looking at how the field is populated in the database, recommend shortening this to "Notes about the emulator", or if you feel it's important to clarify that's it's not displayed: "Private notes about the emulator".